### PR TITLE
[circle-mpqsolver] Revise BisectionSolver

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit uses quantizer context in constructor of BisectionSolver instead of  passing them directly.

Draft: https://github.com/Samsung/ONE/pull/11849
Related: https://github.com/Samsung/ONE/issues/11374

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: compiler/circle-mpqsolver/src/core/Quantizer.h [hunk 1]
```diff
@@ -48,8 +48,8 @@ class Quantizer
   {
     std::string output_model_dtype;
     std::string granularity;
-    std::string input_type;
-    std::string output_type;
+    std::string input_type = "uint8";
+    std::string output_type = "uint8";
```
Comment (seanshpark) [inline compiler/circle-mpqsolver/src/core/Quantizer.h:52]: IMHO, these two lines should be split to another PR

Comment (seanshpark) [reply compiler/circle-mpqsolver/src/core/Quantizer.h:52]: Q) why `output_model_dtype` and `granularity` not given default values?

Comment (stamalakhov) [reply compiler/circle-mpqsolver/src/core/Quantizer.h:52]: > Q) why `output_model_dtype` and `granularity` not given default values?

Changing `input_type` and `output_type` values was needed for tests, and `granularity` and `output_model_dtype` is hardcoded now in `master`, so setting `granularity` and `output_model_dtype` is not necessary.
I'll move these changes to next PR. Thank you.

Comment (stamalakhov) [reply compiler/circle-mpqsolver/src/core/Quantizer.h]: > IMHO, these two lines should be split to another PR

done

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM
